spec/models/info_request_spec.rb

```diff
@@ -1677,13 +1677,13 @@
     end
 
     context 'email with a broken id and an intact idhash' do
-      let(:email) { "request-123ab-#{ info_request.idhash }@example.com" }
+      let(:email) { "request-ab123-#{ info_request.idhash }@example.com" }
```

Comment:
-1 to this commit. This doesn't fix anything, only hides the problem.

A solution to this is something like https://github.com/mysociety/alaveteli/issues/4996#issuecomment-444850057.

As https://github.com/mysociety/alaveteli/issues/4996 wasn't introduced in this PR, we'll fix it independently to avoid confusion.